.github/workflows/release_pypi.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and release lightly
+
+on:
+  workflow_dispatch:
+# TODO(Philipp, 03/23): Enable me after proper testing.
+#  release:
+#    types: [created]
```

Comment:
Are you sure we should use `created`? From the docs it seems to me that we should use `published`.

Reply:
You might be right although I'm not 100% what the difference is. I'll change it.